### ISSUE TITLE
Howling Fjord quest 'Absholutely... Thish Will Work' 

### ISSUE THREAD

**Issue by Raulfin** [dalaranwow/dalaran-wow#4408]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)
Howling Fjord quest 'Absholutely... Thish Will Work' 
http://www.wowhead.com/quest=11330/absholutely-thish-will-work
https://www.youtube.com/watch?v=ZN-XDtDdSuE

(also as an aside the gate to the prison should be in the down position until used, it currently sits in the open position until used.)

**Current behaviour**: Using the item whith the prisoner targeted does nothing. No credit is given.

**Expected behaviour**: When used the prisoner should drink the item stand and yell, and quest progress made. As shown in the video.

**Steps to reproduce the problem**:

1. Talk to Peppy Wrongnozzle to accept the quest, after completing Forsaken Intel- http://www.wowhead.com/quest=11328/mission-forsaken-intel
2. enter the prison, target the prisoner and use the item.
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Teridan-dal-wow** [comment]
Can not confirm that, works fine for me.

**Comment by Rushor** [comment]
on tc server or on dalaran-wow? @Teridan-dal-wow

**Comment by Rushor** [comment]
hm however on dalaran-ptr it was not working, i will add the script for it

**Comment by Teridan-dal-wow** [comment]
On tc server, i can't test it on dalaran as i don't have a char that can do it.

**Comment by Rushor** [comment]
DB/Quest:  Absholutely... Thish Will Work - Add questscript for Westguard Defender and Dragonflayer Vrykul Prisoner

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4408
* commit: 524d00d0edf2720a2b0c3f3ac2f89d6eb87eac73

![GitHub Logo](http://i.imgur.com/6eKb6Mz.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
yeah on several things tc and dalaran-wow(ptr) are different. but you will get your ptr-test-account soon

**Comment by Pellaeon83** [comment]
Can the fix be applied to the Kirin-Tor realm as well please? Quest at the moment impossible to finish and without it all quests for Skorn and Gjalerbron are unobtainable (not to mention some chains in Fort Wildevar) which is 20+ quests.
Please add the fix on Kirin-Tor realm :)

**Comment by trekskel** [comment]
Hi, 
This quest is still not working on kirin-tor realm. Could you please fix it ?

thanks.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
